tailscale/DOCS.md

```diff
@@ -70,6 +70,8 @@ advertise_routes:
   - 192.168.1.0/24
   - fd12:3456:abcd::/64
 funnel: false
+healthcheck_offline_timeout: 110
+healthcheck_restart_timeout: 3600
```

Comment:
I really don't want to keep add options like this.

People shouldn't have to worry about this stuff.

../Frenck

Reply:
Hardwire 5 minutes and 1 hour? Based on my logs, 2-3 minutes long dropouts occure approx. daily.

It also means that healthcheck will be live after update. I'm just a bit afraid that a "regular" restart can ruin something. Maybe DNS. I was never able to configure HA to use TS's DNS, so I can't test it, it's just a wild guess.